repolint.json

```diff
@@ -144,12 +144,13 @@
             "**/*.cc",
             "**/*.cpp",
             "**/*.h",
-            "**/*.ts"
+            "**/*.ts",
+            "**/*.sh"
           ],
           "skip-paths-matching": {
             "patterns": [
               "babel.config.js",
-              "build/*",
+              "build\/",
               "jest.config.js",
               "node_modules/*",
               "types/*"
```

Comment:
Hey can you also fix these others while you're at it? The types one might cause issues in the future. e.g. for the js files do we need to escape the period? And the once with wildcard, apply same fix? Thanks